### ISSUE TITLE
Repeat/Loop function

### ISSUE THREAD

**Issue by brdppl** [vmudigal/ngx-audio-player#47]
First, this is a great audio player for Angular!
But, is it possible to implement the repeat function?
Thanks!

**Comment by Abdullah0991** [comment]
For anyone want to implement this feature manually:

In template:
`<mat-basic-audio-player #audio_player (trackEnded)="playAudioFileEnded($event)"></mat-basic-audio-player>`

In ts:
```
@ViewChild('audio_player', { static: false })
audioPlayer: MatBasicAudioPlayerComponent;

playAudioFileEnded() {
  this.audioPlayer.play();
}
```

**Comment by shajuMurugan** [comment]
Hi I need to hide the play next and prev is possible??

**Comment by vmudigal** [comment]
Feature available in v8.1.4, v9.2.3, v10.1.4 and v11.0.4.
